### DISCUSSION TITLE
temporarily remove a piece?

### DISCUSSION THREAD

**Discussion by akamegakira** [SebLague/Chess-Challenge#288] [Q&A]
does anyone have a function that temporarily removes a piece from the board, or something equivalent to that?

**Comment by mcthouacbb** [comment]
A few options come to mind(there's probably more ways to do this, but this is what I could think of)

1. Extract the FEN of the board, remove the piece from the FEN, and create a new board with Board.CreateBoardFromFen()(Will be slow)
2. Use bitboards to "emulate" the removal of the piece without actually removing it. This might work better if you only need a few properties of the resulting board, and not the whole board.
3. Pretend the piece isn't there. If you don't need to access any board apis then this method could work.
4. Submit an issue asking for the feature(probably won't happen, but you can try)
